packages/presentation/lib/screen/home_bloc.dart

```diff
@@ -3,7 +3,6 @@ import 'package:presentation/base/bloc_base.dart';
 import 'package:presentation/screen/home_data.dart';
 import 'package:injectable/injectable.dart';
 
-// @singleton
 abstract class HomeBlocInterface extends Bloc {
```

Comment:
u can name abstract class HomeBloc
and it's impl _HomeBloc as u shouldn't have an access to _HomeBloc from anywhere outside the file